package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "2.0.24",
+  "version": "2.0.26",
```

Comment:
This is weird. Since you rebased, this should show a change from "25" to "26". ???

(Anyway it's correct so don't worry about it.)